lib/prepare_security.js

```diff
@@ -16,31 +25,63 @@ export default class SecurityReleaseSteward {
     });
 
     const req = new Request(credentials);
-    const create = await cli.prompt(
+    const release = new PrepareSecurityRelease(req);
+    const releaseDate = await release.promptReleaseDate(cli);
```

Comment:
We have `promptReleaseDate` and `cli.prompt(
      'Create the vulnerabilities.json?',
      { defaultAnswer: true });`

we should make a pattern here, either you remote the `promptReleaseDate` or you create `release.promptVulnerabilitiesJSON()`